cmd/aws-iam-authenticator/server.go

```diff
@@ -139,12 +123,16 @@ func init() {
 		"master is the URL to the api server")
 	viper.BindPFlag("server.master", serverCmd.Flags().Lookup("master"))
 
-	serverCmd.Flags().String(
-		"address",
+	serverCmd.Flags().String("address",
 		"127.0.0.1",
 		"IP Address to bind the server to listen to. (should be a 127.0.0.1 or 0.0.0.0)")
 	viper.BindPFlag("server.address", serverCmd.Flags().Lookup("address"))
 
+	serverCmd.Flags().StringSlice("backend-mode",
+		[]string{"ConfigMap", "File"},
```

Comment:
is this missing `crd`? Also should these be Capitalized? Maybe stored and a `const` and referenced?

Reply:
yes, added crd & replaced with const